plugin/trino-hive/src/main/java/io/trino/plugin/hive/TrinoViewHiveMetastore.java

```diff
@@ -193,6 +193,27 @@ public void updateViewComment(ConnectorSession session, SchemaTableName viewName
         replaceView(session, viewName, view, newDefinition);
     }
 
+    public void updateViewColumnComment(ConnectorSession session, SchemaTableName viewName, String columnName, Optional<String> comment)
```

Comment:
Please extract `io.trino.plugin.iceberg.catalog.hms.TrinoHiveCatalog#updateViewColumnComment` in a preparatory commit.